docs/docsite/versions.yaml

```diff
@@ -0,0 +1,20 @@
+package:
```

Comment:
This name is rather misleading. Both are packages. `ansible-core` is a distribution package that does not bundle heavy stuff.